### PR TITLE
environment variables & reorganization cli commands content

### PR DESCRIPTION
**Pull request by juanmaguitar**

### CODE REVIEW

File: docs/frontity-cli/README.md [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
+#### Create commands
+
+These commands will allow you to either create a Frontity project or a Frontity package
+
+* [`create`](./create-commands#create)
+* [`create-package`](./create-commands#create-package)
+
+#### Run commands
+
+These commands will allow you run a Frontity project in development or production mode
```
[the remark this answers is not in the document]
Comment (juanmaguitar) [reply docs/frontity-cli/README.md:26]: Well, I think it fits in the sense that it generates the code that will be executed via `npx frontity serve` or as serverless functions
Any suggestions regarding this?

File: docs/frontity-cli/create-commands.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+This argument sets the _name_ of your Frontity package.
+The `create-package` command will create a folder named `[package-name]` under `packages`.
+It will also add the proper dependency in the `package.json` of your Frontity project
+
+#### **`[options]`**
+
+| Option | Description |
+| :---: | :--- |
+| `-n <value>`, `--namespace <value>` | Sets the [namespace](https://docs.frontity.org/learning-frontity/namespaces) for this package |
+| [`-n`, `--no-prompt`](#) | Skips prompting the user for options.  </br> Related environment variable: [`FRONTITY_NAME`](./environment-variables#FRONTITY_NAME)  </br> Read more about this option [here](#)|
```
[the remark this answers is not in the document]
Comment (juanmaguitar) [reply docs/frontity-cli/create-commands.md:107]: The thing is w/ gitbook we cannot preview the code generated and I cannot know the final ID's that gitbook will generate from these headers
This is one of the problems I would like to solve with a new documentation platform (being able of preview the final result before publish)
For now, I think the only option is doing this in a 2 step process → Once it's published I will commit and merge directly to merge the right links

File: docs/frontity-cli/create-commands.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+It will also be the name of the folder that this command will create for you with the files of your Frontity project inside
+
+#### **`[options]`**
+
+| Option | Description |
+| :---: | :--- |
+| `-h <theme>`, `--theme <theme>` | The theme to use |
+| `-t`, `--typescript` | Adds support for TypeScript |
+| `-c`, `--use-cwd` | Generates the project in the current directory |
+| [`-n`, `--no-prompt`](#) | Skips prompting the user for options.  </br> Related environment variable: [`FRONTITY_NAME`](./environment-variables#FRONTITY_NAME)  </br> Read more about this option [here](#)|
```
[the remark this answers is not in the document]
Comment (juanmaguitar) [reply docs/frontity-cli/create-commands.md:35]: Same than https://github.com/frontity/docs/pull/185#discussion_r470176646

File: docs/frontity-cli/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (juanmaguitar) [reply docs/frontity-cli/README.md:26]: Would it make sense to have?
- Run Commands → `dev` & `serve`
- Build Commands → `build`
- Create Commands → `create` & `create-package`

[the remark this answers is not in the document]
Comment (michalczaplinski) [reply docs/frontity-cli/README.md]: Sorry, I havent replied earlier @juanmaguitar yeah, I guess that is fine 🙂

**Pull request merged**